python/core/auto_generated/vectortile/qgsmapboxglstyleconverter.sip.in

```diff
@@ -487,7 +496,7 @@ Takes a QColor object and returns HSLA components in required format for QGIS :p
    This is private API only, and may change in future QGIS versions
 %End
 
-    static QString interpolateExpression( double zoomMin, double zoomMax, double valueMin, double valueMax, double base, double multiplier = 1 );
+    static QString interpolateExpression( double zoomMin, double zoomMax, QVariant valueMin, QVariant valueMax, double base, QgsMapBoxGlStyleConversionContext &context, double multiplier = 1 );
```

Comment:
Inserting the new parameter before an existing one is an api break -- can you move context to sit after multiplier?

Reply:
Done. 
Shall I also remove the comment 'This is private API only , and may change in future QGIS versions' ?